src/api/prop.js

```diff
@@ -26,9 +26,10 @@ export const boolean = create({
 });
 
 export const number = create({
-  coerce: alwaysUndefinedIfEmptyOrNumber,
-  deserialize: alwaysUndefinedIfEmptyOrNumber,
-  serialize: alwaysUndefinedIfEmptyOrNumber
+  default: 0,
+  coerce: alwaysZeroIfNotANumberOrNumber,
```

Comment:
What happens if you don't change `undefined` to `0`? I would think that the default would still be `0`, right?